### DISCUSSION TITLE
Error trying to run Linux Quickstart on an arm processor

### DISCUSSION THREAD

**Discussion by ivanrulik** [Yaskawa-Global/YMConnect#16] [Q&A]
Hi everyone. 

I'm trying to follow the steps over [here](https://developer.motoman.com/en/YMConnect/LinuxQuickStart)  to run the quick start example on an arm based Linux computer but I get the following error:

```bash
~/Desktop/HelloYMConnect $ cmake --build .
[ 50%] Linking CXX executable HelloYMConnect
/usr/bin/ld: libYMConnect_D.so: error adding symbols: file in wrong format
collect2: error: ld returned 1 exit status
gmake[2]: *** [CMakeFiles/HelloYMConnect.dir/build.make:98: HelloYMConnect] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:83: CMakeFiles/HelloYMConnect.dir/all] Error 2
gmake: *** [Makefile:91: all] Error 2
```

Is there a chance to get the library files for arm processors, or any way to compile the library manually?

**Comment by ted-miller** [comment]
Hello @ivanrulik 
At this time, we only support x64 processors and do not have support for ARM. We will consider this CPU architecture for a possible future update.
